### PR TITLE
Navbar and footer

### PR DESCRIPTION
**Pull request by JoseLopezSolis**
# What did we implement in this PR?

- We Implements the header and footer design.
- The design implemented is responsive for all devises.
- We implement real customer information.
## Screenshots
image

image

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+ @import "./layout/navbar";
+
+
+
+
+ body{
+  padding: 0!important;
+  margin:0!important;
+  font-family: var(--font-family-main) !important;
+ }
```
Comment (JuanequeX) [inline app/assets/stylesheets/application.scss]: - Add a space between the class and your curly brace
- remove the `!importants` totally is a bad practice used its.

File: app/assets/stylesheets/components/_top-message.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.top-message-size{
+  font-size: .9rem !important;
+  letter-spacing: 0.05rem !important;
+  background-color: var(--black-background) !important;
```
Comment (JuanequeX) [inline app/assets/stylesheets/components/_top-message.scss]: - sort alphabetically this block
- remove impotants
- add space in your class with the curly brace
- add a blank line at the end of this file

File: app/assets/stylesheets/config/variables.scss [hunk 3]
```diff
@@ -0,0 +1,6 @@
+@import url('https://fonts.googleapis.com/css2?family=Assistant:wght@200;300;400;500;600;700;800&display=swap');
+
+:root{
+  --font-family-main: 'Assistant', sans-serif;
+  --black-background: #181818;
```
Comment (JuanequeX) [inline app/assets/stylesheets/config/variables.scss:5]: in scss the correct way to declare variables is with `$` like this:
- `$black-background: #181818;`

File: app/assets/stylesheets/layout/_footer.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
+.background-footer{
+  background-color: var(--black-background)!important;
```
Comment (JuanequeX) [inline app/assets/stylesheets/layout/_footer.scss]: the way to call your color in scss should be like this:
- `$black-background;`
- Remove the important.

File: app/assets/stylesheets/layout/_navbar.scss [hunk 5]
```diff
@@ -0,0 +1,24 @@
+.logo-size {
+  width: 9rem !important;
```
Comment (JuanequeX) [inline app/assets/stylesheets/layout/_navbar.scss]: please remove the important

File: app/assets/stylesheets/layout/_navbar.scss [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+.active{
+  color: inherit;
+  text-decoration: underline 0.10rem rgb(0, 0, 0);
+}
+
+.color-icon{
+  font-size: 1.3rem;
+  color: rgb(0, 0, 0) !important;
+  cursor: pointer;
```
Comment (JuanequeX) [inline app/assets/stylesheets/layout/_navbar.scss]: sort alphabetically

File: app/helpers/home_helper.rb [hunk 7]
```diff
@@ -0,0 +1,2 @@
+module HomeHelper
+end
```
Comment (JuanequeX) [inline app/helpers/home_helper.rb:2]: if this helper hasn't any propose in this feature please remove it.

File: app/views/partials/layout/_navbar.html.erb [hunk 8]
```diff
@@ -0,0 +1,40 @@
+<div class="container-flouid">
+  <nav class="navbar navbar-expand-lg bg-light shadow-sm p-3 mb-5 bg-body-tertiary rounded">
+    <div class="container">
+      <a class="navbar-brand" href="/">
+      <%= image_tag("monkeyHats.jpg", class: "rounded-circle logo-size mx-auto") %>
```
Comment (JuanequeX) [inline app/views/partials/layout/_navbar.html.erb:5]: totally is better use images in the `webp` format this bring a better performance in the SEO and in the `lighthouse` tests

**Review by JuanequeX — CHANGES_REQUESTED** [review]
I'll recommend that the base language for this project are in English because this is one approach to work in `brightcoders`

**Review by claunicole — APPROVED** [review]

**Review by JuanequeX — APPROVED** [review]

**Pull request merged**